frontend/stylesheets/_utils.less

```diff
@@ -122,3 +122,7 @@ td.nowrap * {
 .min-height-0 {
   min-height: 0;
 }
+
+.text-center {
```

Comment:
This is not used in the PR. Do you still want to merge it?

Reply:
I noticed that the spinner in question was centered by using a `text-center` class which didn't exist (which didn't work obviously). That's why, I added this class.

Reply:
Ah. Good point. Maybe a Bootstrap leftover: https://getbootstrap.com/docs/5.0/utilities/text/#text-alignment